app/models/notification.rb

```diff
@@ -4,6 +4,8 @@ class Notification < ApplicationRecord
   belongs_to :recipient, class_name: 'User'
   belongs_to :notifiable, polymorphic: true
 
+  alias_method :user, :recipient
```

Comment:
This seems to only be used by the presenter? I see how it may be interesting to have a common interface between Notification and Activity if we're going to create the general presenter, but that refactor should be part of the extraction of the presenter. 

In this PR I'd stick to :recipient throughout the code